src/main/java/seedu/address/logic/commands/profile/ProfileCommand.java

```diff
@@ -11,13 +11,17 @@ public abstract class ProfileCommand extends Command {
 
     public static final String COMMAND_WORD = "profile";
 
-    public static final String PROFILE_FORMAT = "Format: " + COMMAND_WORD + " -option";
+    public static final String PROFILE_FORMAT = "Format: " + COMMAND_WORD + " -option\n";
 
-    public static final String OPTION_UNKNOWN = "That is not a valid option flag. "
-            + " Please use one of the following valid flags: "
-            + PREFIX_OPTION + AddProfileCommand.COMMAND_OPTION + ","
+    public static final String VALID_FLAGS = "Please use one of the following valid flags: "
+            + PREFIX_OPTION + AddProfileCommand.COMMAND_OPTION + ", "
+            + PREFIX_OPTION + DeleteProfileCommand.COMMAND_OPTION + ", "
+            + PREFIX_OPTION + EditProfileCommand.COMMAND_OPTION + ", "
+            + PREFIX_OPTION + ViewProfilesCommand.COMMAND_OPTION + ", "
             + PREFIX_OPTION + FindProfileCommand.COMMAND_OPTION + ".";
```

Comment:
Minor, but I think we were going for alphabetical order, so perhaps we could swap the `View` and `Find`.